deploy/docker-compose.yaml

```
@@ -69,6 +69,17 @@ services:
       interval: 5s
       timeout: 1s
       retries: 10
+  mongo-express:
+    image: mongo-express
+    environment:
+      - ME_CONFIG_MONGODB_SERVER=mongodb
```

Zmienić envy na ten format: 
```suggestion
   ME_CONFIG_MONGODB_SERVER: mongodb
```